packages/solicitudes/src/adapters/db/index.js

```diff
@@ -0,0 +1,12 @@
+// External dependencies
+const { drivers, queries } = require('@siiges-services/core');
+
+const { Solicitud } = drivers.sequelize.models;
```

Comment:
You can use models here
```suggestion
const { models, queries } = require('@siiges-services/core');

const { Solicitud } = models;
```

Reply:
Done